src/test/java/capstone/mfslbackend/service/TeamServiceTest.java

```diff
@@ -46,18 +46,21 @@ public void testCreateTeamsInLeague_SuccessTeamDoesntExist() throws IOException
         String leagueId = "123";
         String season = "2023";
 
-        Long teamId = 1L;
+        long teamId = 1L;
         String teamName = "test";
+        String theupdate = "lol";
+        Boolean winner = true;
+        String code = "3029";
+        String country = "Portugal";
+        Integer founded = 2;
+        Boolean isNational = true;
         // Mock the ApiService to return a list of teams
         TeamsContainer teamsContainer = new TeamsContainer();
         List<TeamsResponse> teamsResponses = new ArrayList<>();
+        //long id, String name, String logo, String update, Boolean winner, String code, String country, Integer founded, Boolean national
+        TeamResponse teamResponse1 = new TeamResponse(teamId, teamName, "lol", "yes", true, code, country, founded, isNational);
```

Comment:
For the specific call the backend makes here, the things you set to lol yes true, code, country, founded and national all just get ignored because we dont keep them, so its not necessary to add them to the test, they dont get transferred into anything